doc/classes/SceneTree.xml

```diff
@@ -18,8 +18,9 @@
 			<param index="0" name="group" type="StringName" />
 			<param index="1" name="method" type="StringName" />
 			<description>
-				Calls [param method] on each member of the given group. You can pass arguments to [param method] by specifying them at the end of the method call. If a node doesn't have the given method or the argument list does not match (either in count or in types), it will be skipped.
-				[b]Note:[/b] [method call_group] will call methods immediately on all members at once, which can cause stuttering if an expensive method is called on lots of members.
+				Calls [param method] on each node inside this tree added to the given [param group]. You can pass arguments to [param method] by specifying them at the end of this method call. Nodes that cannot call [param method] (either because the method doesn't exist or the arguments do not match) are ignored. See also [method set_group] and [method notify_group].
+				[b]Note:[/b] This method acts immediately on all selected nodes at once, which may cause stuttering in some performance-intensive situations.
+				[b]Note:[/b] In C#, [param method] must be in snake_case when referring to built-in Godot methods.
```

Comment:
Add a recommendation to use the cached `StringName` like we have in the `Object.call` documentation.

https://github.com/godotengine/godot/blob/ba54c34551d1bda5139515d74e7d614ccbc43cea/doc/classes/Object.xml#L371